community/cypher/cypher/src/main/java/org/neo4j/cypher/javacompat/ExecutionResult.java

```diff
@@ -281,9 +284,19 @@ public void remove()
     }
 
     @Override
+    public void accept( ResultVisitor visitor )
```

Comment:
This needs to be moved deeper into the guts of the execution engine (inside of compiler-2.3) so that we can replace it with something faster from Birk.